ui/shared/CallbackMessage.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Row } from 'reactstrap';
+
+import ErrorMessages from './ErrorMessages';
+
+const CallBackMessage = ({ errorMessage, text }) => (
+  <div className="pt-5">
+    {errorMessage ? (
+      <ErrorMessages failureMessage={errorMessage} />
+    ) : (
+      <Row className="justify-content-center">
+        <p className="lead text-center">{text}</p>
+      </Row>
+    )}
+  </div>
+);
+
+CallBackMessage.propTypes = {
+  errorMessage: PropTypes.string,
+  text: PropTypes.string.isRequired,
+};
+
+CallBackMessage.defaultProps = {
+  errorMessage: '',
+};
+
+export default CallBackMessage;
```

Comment:
We can probably rename the component to match the file name (lower case b).

Reply:
Good spot.